src/H5private.h

```diff
@@ -363,7 +363,6 @@
  */
 #define SUCCEED 0
 #define FAIL    (-1)
-#define UFAIL   (unsigned)(-1)
```

Comment:
Picking 2^32 as the "bad" value isn't going to work so hot when assigned to size_t on a 64-bit platform.